node/lib/package/LocalPackageSource.ts

```diff
@@ -278,7 +284,7 @@ export class LocalPackageSource extends PackageSource {
                     let schemaModulePath: string = schemaId + "/" + schemaId;
                     schemaModulePaths.push(packageJson.name + "/" + schemaModulePath);
                     schemaInfos.push({
-                        moduleName: schemaModulePath
+                        moduleName: schemaModulePath,
```

Comment:
I don't get why you had to add a comma... tslint? Just proving that I'm reading... though I don't really care.